neuroanalysis/util/mies_nwb_parsing.py

```diff
@@ -0,0 +1,192 @@
+from collections import OrderedDict
```

Comment:
Organizational thoughts: probably neuroanalysis.data should be a subpackage, and then perhaps neuroanalysis.data.mies_nwb and .acq4 below that? (then mies_nwb_parsing would go into data.mies_nwb)